frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/CheckUserExistsHandlerTest.java

```diff
@@ -126,6 +128,61 @@ void setup() {
         reset(authenticationService);
     }
 
+    @Test
+    void shouldReturn200WithLockInformationIfUserExistsAndMfaIsAuthApp() throws Json.JsonException {
```

Comment:
Is there a way we can tidy up these tests a bit? I'm thinking:
a) is there any common setup that can be pulled out
b) or are there any common helper methods that could be introduced / variables (e.g. the headers stuff is mostly just noise
c) The assertions to check what's in the json response are quite noisy - I generally find it a bit more readable if we just assert on some constructed json

What you've written is very much in the style of a lot of tests in this codebase, but I'd love it if we could do some tidying along the way.

Reply:
As an example, I recently refactored the tests in [here](https://github.com/govuk-one-login/authentication-api/blob/main/frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/ResetPasswordRequestHandlerTest.java) to remove a lot of noise. In general the principle I'm following is to try make it very clear:

1. what the input is that is changing that determines the behaviour of the test
2. what the expected output is given the different input

Reply:
As an example, I recently refactored the tests in [here](https://github.com/govuk-one-login/authentication-api/blob/main/frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/ResetPasswordRequestHandlerTest.java) to remove a lot of noise. In general the principle I'm following is to try make it very clear:

1. what the input is that is changing that determines the behaviour of the test
2. what the expected output is given the different input